### PR TITLE
242 frontend backend finish workout tracking flow

### PR DESCRIPTION
**Pull request by mva919**
# What type of PR is this? (Check all that apply)

- [ ] ✨**feature**: Introduces completely new code or new features.
- [ ] 🐛**fix**: Implements changes that fix a bug. Ideally, reference an issue if present.
- [ ] ♻️**refactor**: Includes any code-related change that is neither a fix nor a feature.
- [ ] ✅**build**: Encompasses all changes related to the build of the software, including changes to dependencies or the addition of new ones.
- [ ] ⚡️**test**: Pertains to all changes regarding tests, whether adding new tests or modifying existing ones.
- [ ] 🚰**ci**: Involves all changes related to the configuration of continuous integration, such as GitHub Actions or other CI systems.
- [ ] 📚**docs**: Includes all changes to documentation, such as README files, or any other documentation present in the repository.
- [ ] 🗑️**chore**: Captures all changes to the repository that do not fit into the above categories.

# Description

What did you change? How did you change it?

# Tests

How was this tested?

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [ ] Manual tests
- [ ] Tests were NOT needed
- [ ] Other (explain below)

# [Optional] Screenshots

# Documentation

- [ ] Added to README.me
- [ ] Seperate document
- [ ] NO documentation needed

Link to external documentation:

# [Optional] AI Usage

# [Optional] Are there any post-deployment tasks we need to perform?

### CODE REVIEW

**Comment by taylorfinelli** [comment]
🪨

File: apps/expo/src/app/(dashboard)/create-new-workout.tsx [hunk 1]
```diff
@@ -16,9 +18,26 @@ const styles = StyleSheet.create({
...
+
   return (
     <SafeAreaView style={styles.container}>
-      <NavBar center='Exercises' right='Next' />
+      <NavBar
+        center='Exercises'
+        right={
+          <Button color='icon' disabled={!selectedExercises.length} onPress={handleStartWorkout}>
+            <Text numberOfLines={1} style={{ color: '#CACACA', fontSize: 16 }}>
+              {selectedExercises.length === 0 ? 'Add' : `Add(${selectedExercises.length})`}
```
Comment (elliotwesoff) [inline apps/expo/src/app/(dashboard)/create-new-workout.tsx:36]: Should there be a space on the `else` of the conditional?

**Review by elliotwesoff — APPROVED** [review]
LGTM

**Review by thienguen — COMMENTED** [review]
I tested and its fine, ready to go with this

**Pull request merged**